docs/community_resources/installing-on-mac-os-and-windows.md

```diff
@@ -123,21 +128,17 @@ The configuration requires providing the following parameters:
 
     You can omit this step. If you do not create a database now, it will be created automatically in the next step.
 
-Create a database. Run the following command inside MySQL Shell:
+Create a database by running the following command inside MySQL Shell:
 
 ``` bash
 CREATE DATABASE ezplatform CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_520_ci;
 ```
 
 ## Install eZ Platform
 
-Before executing the following command make sure that the user set during `composer install` has sufficient permissions.
-
-Install eZ Platform with:
+Before executing the following command, ensure that the user set during `composer install` has sufficient permissions.
 
-``` bash
-composer ezplatform-install
-```
+Install eZ Platform by running: `composer ezplatform-install`.
```

Comment:
I'd leave the command in a separate code block. True, it is only one line, but it's one of the two main commands on this page and should be emphasised.